### ISSUE TITLE
New Prototype

### ISSUE THREAD

**Issue by pufflegamerz** [heb12/heb12-android#7]
The new prototype is out! I decided to make it in MIT's App Inventor since it's a lot easier (and it will soon support iOS).

Online Emulator: https://appetize.io/app/b6dudg4aqpzvkzauhb5123bka4?device=nexus5&scale=75&orientation=portrait&osVersion=7.1

**Comment by MasterOfTheTiger** [comment]
Upload the APK. :)

**Comment by pufflegamerz** [comment]
It's in the repository:
https://github.com/heb12/heb12-android/blob/master/Heb12.apk

**Comment by MasterOfTheTiger** [comment]
Okay. Let's divide these into different branches. Make a new branch (called alpha or something) and upload the code into there (but not as a zip).

**Comment by pufflegamerz** [comment]
Sounds good.

**Comment by pufflegamerz** [comment]
I found the entire Bible in a .json file: https://github.com/thiagobodruk/bible/tree/master/json

**Comment by MasterOfTheTiger** [comment]
Awesome! See [openbibles](https://gitlab.com/masterofthetiger/openbibles) for something you should port and use for this app. It supports quite a few translations. It would be very useful.

**Comment by MasterOfTheTiger** [comment]
@pufflegamerz The .aia version in the alpha folder is old.

**Comment by MasterOfTheTiger** [comment]
Remove all the spaces from the book names for it to work.

**Comment by pufflegamerz** [comment]
Do you think this app is ready for the app store yet?

**Comment by MasterOfTheTiger** [comment]
No. Sorry, but it needs a bunch more work.

**Comment by MasterOfTheTiger** [comment]
Wait, I didn't user 1.2 yet.

**Comment by pufflegamerz** [comment]
Could you remove the "It does not work." now that I have made a successful prototype?

**Comment by MasterOfTheTiger** [comment]
Yes @pufflegamerz, I did it, but can you upload an .apk and .aia for 1.2?

**Comment by pufflegamerz** [comment]
Done.

**Comment by pufflegamerz** [comment]
Do you have any ideas on what we can make the app on? Android Studio won't work for me and App Inventor is limited.

**Comment by MasterOfTheTiger** [comment]
@pufflegamerz I would recommend real coding. Android Studio is a tool for it, but you can make Android apps the same way as Android Studio without that specific IDE. It may be a bit harder though. You need to download the Android SDK any way. 

I personally deleted Android Studio because it took up too much space on my computer. But if I have to I can reinstall it.

**Comment by pufflegamerz** [comment]
I have finally found something to make the app in. It's a visual studio extension called Xamarin. I can code the app in C# and design the interface in XML.

**Comment by MasterOfTheTiger** [comment]
@pufflegamerz That's great! I am glad we can progress in the development in this app. However, is C# the best for this?

**Comment by pufflegamerz** [comment]
C# is the easiest language I've found that I can compile apps with.

**Comment by pufflegamerz** [comment]
Why don't you like C#?
